### PR TITLE
Add optional NFS location and functionality for bbb recordings

### PR DESCRIPTION
**Pull request by Davka**
adds the ability to store recordings with scalelite on nfs-share

### CODE REVIEW

**Review by BrutalBirdie — CHANGES_REQUESTED** [review]
First of all, thanks for the PR!
I added some of my thoughts.

The whole recording and scalelite part is total alien to me since I never use BBB recordings, but would be happy about a session showing this in action.

**Review by Davka — COMMENTED** [review]
i think i have done all the change requests

File: tasks/installation.yml [hunk 1]
```diff
@@ -47,3 +47,10 @@
         - docker-compose==1.24.1
       state: "{{ bbb_state }}"
   when: bbb_greenlight_enable | bool or bbb_monitoring_all_in_one_enable | bool
+
+- name: Install nfs-common
+  apt:
+    name:
+      - nfs-common
+    state: latest
+  when: not bbb_disable_recordings | bool
```
[the remark this answers is not in the document]
Comment (Davka) [reply tasks/installation.yml]: Done

File: tasks/main.yml [hunk 2]
```diff
@@ -22,6 +22,9 @@
 
 - import_tasks: dial-in/main.yml
 
+- import_tasks: recordings.yml
+  when: not bbb_disable_recordings | bool and bbb_recordings_nfs_mount | length > 0
+
```
[the remark this answers is not in the document]
Comment (Davka) [reply tasks/main.yml]: Done

**Comment by crosscodr** [comment]
Hi @Davka, this PR needs to be rebased on current HEAD to get rid of ansible-lint-errors and to resolve conflicts.

**Pull request closed without merging**